src/Alias.php

```diff
@@ -10,13 +10,13 @@
 
 namespace Barryvdh\LaravelIdeHelper;
 
-use Closure;
-use ReflectionClass;
 use Barryvdh\Reflection\DocBlock;
 use Barryvdh\Reflection\DocBlock\Context;
+use Barryvdh\Reflection\DocBlock\Serializer as DocBlockSerializer;
 use Barryvdh\Reflection\DocBlock\Tag\MethodTag;
+use Closure;
 use Illuminate\Config\Repository as ConfigRepository;
-use Barryvdh\Reflection\DocBlock\Serializer as DocBlockSerializer;
+use ReflectionClass;
```

Comment:
Laravel (since 6?) finally switched to alphabetical imports, which is reflected in all the files.